15/umbraco-cms/implementation/services/README.md

```diff
@@ -223,6 +223,10 @@ public class HandleUnPublishingHandler : INotificationHandler<ContentUnpublished
 }
 ```
 
+#### Accessing the Published Content Cache via IPublishedContentQuery
+
+When you need to fetch multiple content items by ID, using UmbracoContext.Content is limited because it only allows retrieving one content item at a time. To query multiple items efficiently, you can use IPublishedContentQuery. For more details, see the [IPublishedContentQuery](../../reference/querying/ipublishedcontentquery.md) article.
```

Comment:
```suggestion
When fetching multiple content items by ID, using `UmbracoContext.Content` is limited because it only allows retrieving one item at a time. To query multiple items efficiently, you can use `IPublishedContentQuery`. For more details, see the [IPublishedContentQuery](../../reference/querying/ipublishedcontentquery.md) article.
```